cassandra/cluster.py

```diff
@@ -669,6 +812,24 @@ def __init__(self,
         if connection_class is not None:
             self.connection_class = connection_class
 
+        self.profile_manager = ProfileManager()
+        self.profile_manager.profiles[EXEC_PROFILE_DEFAULT] = ExecutionProfile(self.load_balancing_policy,
+                                                                               self.default_retry_policy,
+                                                                               Session._default_consistency_level,
+                                                                               Session._default_serial_consistency_level,
+                                                                               Session._default_timeout,
+                                                                               Session._row_factory)
```

Comment:
In the next major release, are these Session attributes going to be removed? or the default exec profile will always be constructed that way?